index.js

```diff
@@ -1,64 +1,65 @@
-const Boom     = require('boom')
-const gimme    = require('@articulate/gimme')
-const jwks     = require('jwks-rsa')
-const jwt      = require('jsonwebtoken')
+const axios                 = require('axios')
```

Comment:
https://www.npmjs.com/package/node-fetch is ~50% smaller than axios

Reply:
`jwks-rsa` already brings in `axios`. Otherwise, i would have just went native `https`.